experimental/sev_guest/src/msr.rs

```diff
@@ -174,6 +190,102 @@ pub fn get_cpuid(request: CpuidRequest) -> Result<CpuidResponse> {
     read_msr().try_into()
 }
 
+/// A request for the hypervisor's preferred location for the GHCB page.
+pub struct PreferredGhcbGpaRequest;
+
+impl From<PreferredGhcbGpaRequest> for u64 {
+    fn from(_request: PreferredGhcbGpaRequest) -> Self {
```

Comment:
Why do we ignore the `_request` here and hard-coded return `0x010`?

Reply:
There is no additional data as part of the request, just the request code. This could have been a constant rather than a struct that implements `From`, but I thought this approach keeps the way the requests are used consistent.